윤건수/programmers/LV2_43164/Solution.java

```diff
@@ -0,0 +1,78 @@
+package programmers.LV2_43164;
+
+import java.util.*;
+
+public class Solution {
+
+    Map<String, List<Ticket>> ticketInfos;
+    final static String START_PORT = "ICN";
+    int totalTicketNum;
+    String result;
+
+    public String[] solution(String[][] tickets) {
+
+        ticketInfos = new HashMap<>();
+        totalTicketNum = tickets.length;
+        result = "";
+
+        for(String[] ticket : tickets){
+            String from = ticket[0];
+            String to = ticket[1];
+            Ticket newTicket = new Ticket(to);
+
+            if(!ticketInfos.containsKey(from)){
+                List<Ticket> fromTickets = new ArrayList<>();
+                ticketInfos.put(from, fromTickets);
+            }
+
+            List<Ticket> fromTickets = ticketInfos.get(from);
+            fromTickets.add(newTicket);
```

Comment:
```suggestion
            ticketInfos.computeIfAbsent(from, key -> new ArrayList<>())
                       .add(newTicket);
```

이렇게 변경 가능해 보이는데... IDE 없이 작성해서 오타가 있을 수 있을 것 같네요!!!